package.json

```diff
@@ -51,9 +51,9 @@
     "codelyzer": "^2.0.1",
     "core-js": "^2.4.1",
     "css-loader": "^0.28.0",
-    "extract-text-webpack-plugin": "^2.1.0",
-    "node-sass": "^4.5.2",
-    "npm-run-all": "^4.0.2",
+    "extract-text-webpack-plugin": "^2.1.2",
+    "node-sass": "4.12.0",
+    "npm-run-all": "4.1.5",
```

Comment:
Within libraries like this we should not lock dependencies.

Reply:
Good point.

BTW, do we have a need for both, `package-lock.json` and `yarn.lock` in this project?